service-clients-gen/src/main/java/com/flipkart/poseidon/serviceclients/idl/pojo/Service.java

```diff
@@ -30,6 +30,7 @@
     private Map<String, String> headers;
     private String[] description;
     private String commandName;
+    private String objectMapperClass;
```

Comment:
While writing service idl, this is the only point of reference. Hence add comment here too that the class should be implementing ObjectMapperConfigurer etc